src/main/mw.c

```diff
@@ -419,10 +420,23 @@ void processRx(void)
     }
 
     throttleStatus_e throttleStatus = calculateThrottleStatus(&masterConfig.rxConfig, masterConfig.flight3DConfig.deadband3d_throttle);
+    rollPitchStatus_e rollPitchStatus =  calculateRollPitchCenterStatus(&masterConfig.rxConfig);
 
+    /* In airmode Iterm should be prevented to grow when Low thottle and Roll + Pitch Centered. */
+    /* This is needed to prevent Iterm winding on the ground, but keep full stabilisation on 0 throttl in air */
```

Comment:
This isnt really made for planes as you describe in your example.

2 things:
1) we want full Iterm active during flight unregarding throttle. This means that it does the same on the ground too! This will spool up the motors till it eventually even can take off. So we need something to guard agains that.
2) I looked into various things to do low throttle ground protection on a neat way without interferring much in flight. Every approach I tried just seemed to dirty IMO like timers etc. Just made it more complex.
So I looked at the actual usage in flight. I realised that 0 throttle + roll pitch centered is pretty much a scenario that never occurs. And when it occurs  it is for a very short term....so just temporary constraining the Iterm is fine.

Any other clean approach is welcome, but make it based on real life situation.

I was considering to add yaw as centered check too? What do you think?